concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -110,6 +110,7 @@ datadog_api_key: ${datadog_api_key:-}
 datadog_app_key: ${datadog_app_key:-}
 enable_datadog: ${ENABLE_DATADOG}
 enable_cve_notifier: ${ENABLE_CVE_NOTIFIER:-false}
+enable_cve_monitor: ${ENABLE_CVE_MONITOR:-false}
```

Comment:
I vote for having only one. I doubt we will enable one without enabling the other.

BUT, maybe you can parametrise the support email instead. In dev/staging you can use the alert maillists. This way, in development, we don't spam support.